### ISSUE TITLE
Cannot redeclare static $defaultName as non static in Make Command

### ISSUE THREAD

**Issue by pmartelletti** [laravel/homestead#681]
Please note that the Homestead issue tracker is reserved for bug reports and enhancements. We are not always able to debug Vagrant, Provider or Operating System issues, but will do our best to help. Thank you!

## Versions
  - Vagrant: 1.9.0
  - Provider: Virtualbox
  - Homestead: 6.2.2

## Host operating system
macOs Sierra 10.12

## Homestead.yaml
Not relevant, as the issue is generated when trying to create this file

## Expected behavior
Homestead.yml file should be created

## Actual behavior
```
Fatal error: Cannot redeclare static Symfony\Component\Console\Command\Command::$defaultName as non static Laravel\Homestead\MakeCommand::$defaultName in /Users/pablomartelletti/Software/frts-link/vendor/laravel/homestead/src/MakeCommand.php on line 13
```

This is fixed if you change the property `$defaultName` to `$name` in the make command, but not sure if this will break anything else.

## Steps to reproduce
1. Install homestead as dev dependency for your project
2. Run `php vendor/bin/homestead make`
3. Error will be shown.

**Comment by svpernova09** [comment]
What version of PHP are you using? I'm unable to reproduce this on a clean folder:

```
┌─(joeferguson@halonine)(~)─────────────────────────────────────(en0:192.168.1.94/24 vboxnet0:192.168.10.1/24)─┐
└─(08:03:49)─ mkdir test                                                                                               ─(10/03/17)─┘

┌─(joeferguson@halonine)(~)─────────────────────────────────────(en0:192.168.1.94/24 vboxnet0:192.168.10.1/24)─┐
└─(08:03:52)─ cd test                                                                                                  ─(10/03/17)─┘

┌─(joeferguson@halonine)(~/test)────────────────────────────────(en0:192.168.1.94/24 vboxnet0:192.168.10.1/24)─┐
└─(08:03:53)─ composer require laravel/homestead                                                                       ─(10/03/17)─┘
Using version ^6.2 for laravel/homestead
./composer.json has been created
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 7 installs, 0 updates, 0 removals
  - Installing symfony/yaml (v3.3.9): Loading from cache
  - Installing symfony/process (v3.3.9): Loading from cache
  - Installing psr/log (1.0.2): Loading from cache
  - Installing symfony/debug (v3.3.9): Loading from cache
  - Installing symfony/polyfill-mbstring (v1.5.0): Loading from cache
  - Installing symfony/console (v3.3.9): Loading from cache
  - Installing laravel/homestead (v6.2.2): Loading from cache
symfony/console suggests installing symfony/event-dispatcher ()
symfony/console suggests installing symfony/filesystem ()
Writing lock file
Generating autoload files

┌─(joeferguson@halonine)(~/test)────────────────────────────────(en0:192.168.1.94/24 vboxnet0:192.168.10.1/24)─┐
└─(08:04:05)(127)─ ./vendor/bin/homestead make                                                                         ─(10/03/17)─┘
Homestead Installed!
```

**Comment by pmartelletti** [comment]
php 7.1. 

Just try it, and n  a  new installation is working (it installs symfony/console 3.3) , but in my project, for some reason (probably some vendor requirement) , I'm using 3.4, which introduces this change and breaks the command.

https://github.com/symfony/console/blob/master/Command/Command.php

So, probably composer.json should restrict symfony/console to 3.3, or change the property name in the make command so that is compatible with symfony/console 3.4.

**Comment by pmartelletti** [comment]
The change is in the Changelog.

https://github.com/symfony/console/blob/master/CHANGELOG.md

**Comment by svpernova09** [comment]
Resolved via #682
